README.md

```diff
@@ -1,31 +1,348 @@
 # Hyperion
 
-TODO: Write a gem description
+Hyperion is a Ruby REST client that follows certain conventions
+layered on top of HTTP. The conventions implement best practices
+surrounding versioning, error handling, and crafting an API for
+consumption by third parties. Hyperion provides abstractions that
+make it easy to follow these conventions consistently across
+projects.
 
-## Installation
+This document describes the conventions, then demonstrates how to
+use the API, and finally shows how to test your client-side code.
 
-Add this line to your application's Gemfile:
+## Conventions
+
+### Versioning
+
+_TODO_
```

Comment:
actually, maybe that comment I just made should be addressed in a 'development' or 'contribution' section. This section can talk about the version policy.